interp/constrextern.ml

```diff
@@ -712,28 +711,37 @@ let extended_glob_local_binder_of_decl loc = function
 
 let extended_glob_local_binder_of_decl ?loc u = DAst.make ?loc (extended_glob_local_binder_of_decl loc u)
 
+(* printing option lookup, used several times *)
+let printing_universes () =
+  let open Printoptions in
+  let opts = get_current_options () in
+  opts.printing_universes
```

Comment:
This function could as well live in `Printoptions`.

Reply:
Right, I could put in getter/setters there, and use them in the option declarations.